unittests/domains/ChemistryTester.cpp

```diff
@@ -14,21 +14,24 @@
 #include "cudaq/optimizers.h"
 
 CUDAQ_TEST(GenerateExcitationsTester, checkSimple) {
-
-  auto [singles, doubles] = cudaq::generateExcitations(2, 4);
-
-  EXPECT_EQ(2, singles.size());
-  EXPECT_EQ(1, doubles.size());
-
-  EXPECT_EQ(0, singles[0][0]);
-  EXPECT_EQ(2, singles[0][1]);
-  EXPECT_EQ(1, singles[1][0]);
-  EXPECT_EQ(3, singles[1][1]);
-
-  EXPECT_EQ(0, doubles[0][0]);
-  EXPECT_EQ(1, doubles[0][1]);
-  EXPECT_EQ(2, doubles[0][2]);
-  EXPECT_EQ(3, doubles[0][3]);
+  std::size_t numElectrons = 2;
+  std::size_t numQubits = 4;
+
+  auto [singlesAlpha, singlesBeta, doublesMixed, doublesAlpha, doublesBeta] =
+      cudaq::get_uccsd_excitations(numElectrons, numQubits);
+  EXPECT_TRUE(doublesAlpha.empty());
+  EXPECT_TRUE(doublesBeta.empty());
```

Comment:
Very minor: It'd be nice to have a test case for this (maybe with more spin orbitals and electrons).

Reply:
I have done test for up to 10 electrons and 20 spin molecular orbitals for system like LiH, N2, and H2O and it works. I am happy to test it again.

Reply:
Marwa has given me a 4 electron, 8 qubit case. I will add it here.